packaging/linux/openra.6

```diff
@@ -0,0 +1,68 @@
+.TH OPENRA 6
+.SH NAME
+openra \- An open-source implementation of the Command & Conquer: Red Alert engine using .NET/Mono and OpenGL.
+.SH SYNOPSIS
+.B openra
+[\fB\Game.Mod=\fR\fImodchooser\fR]
+.SH DESCRIPTION
+.B openra
+starts the game.
+.SH OPTIONS
```

Comment:
The bloat can be solved by only extracting settings that have descriptions.  We can also mirror the output to a wiki page, which will address a long-standing documentation request.

As for stability, we've been actively planning on changing the way that servers work (and thereby breaking all of the Server.* options included here) over the last couple of weeks.  If we can't keep this in sync automatically, then could we delay this until after that change has been done?